modules/custom/openy_system/src/Form/OpenyModulesListForm.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\openy_system\Form;
+
+use Drupal\system\Form\ModulesListForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides openy modules installation interface.
+ */
+class OpenyModulesListForm extends ModulesListForm {
+
+  protected $openyPackages = [
+    'OpenY',
+    'OpenY (Experimental)',
+    'YMCA Maryland',
```

Comment:
Is it possible to make this as a config?

Reply:
@podarok I think we will provide more flexible solution for this later, for now I just created very simple page that will be completely rebuild in future.

Reply:
ok